mastering-plone/endpoints.md

```diff
@@ -2,17 +2,17 @@
 
 # Endpoints
 
-````{sidebar}
-Volto chapter
+````{sidebar} Volto chapter
 ```{figure} _static/volto.svg
-:alt: Volto Logo
+:alt: Plone Logo
```

Comment:
Let's check this. The SVG is the Volto logo, but you changed the alt to Plone. I'm not sure what the Plone Marketing team has decided about all the various product logos.

Reply:
Yeah, two things:
- I did copy/paste mistake. Will correct this.
- The Volto-Logo is to be discussed. We are using this for a clear, obvious distinction between backend chapters and frontend chapters. But, yes the Logo of Volto: If the training is regarded as an internal source, then (according to my current state of knowledge) it is correct to speak of 'Volto'. But if the training is regarded also as a showcase, than the term should be avoided and same for the Volto logo. So this is open and should be checked by the marketing team. Any opinions @pbauer?

Reply:
I think it was discussed and consensus to use the term Volto for technical documentation, similar as we use Dexterity, but do not use the logo anymore.

Reply:
Thanks for the hint, @jensens. We have multiple chapters in one of the trainings. Philip picked my idea of a clear visual label where the journey of the trainee is: backend , frontend, … And I think the implementation by sidebar with title and logo of backend/frontend is pointing.

Reply:
Me/We is now searching for a solution and I stay for the moment with the Volto logo. It is temporary, as I / we relaunchers are aware that the volto logo is to be replaced in the training documentation.

Reply:
What about using our Logo guidelines and have something like this in the sidebar for every section.
The text at the right can be replaced by whatever makes sense, like Volto, Classic UI, Backend, Restapi ....
![plone-volto-logo](https://user-images.githubusercontent.com/157140/132678213-1512852f-6dfc-4163-a45c-ad4dc2c89d8a.png)

Reply:
I like the idea to have a Plone logo with the variation ( in form of text) on the right
for labeling the chapters.

Reply:
May I suggest to move this topic 'logo' to a separate issue. For example https://github.com/plone/training/issues/517
and move forward with the relaunch.